app/templates/books/book/pages/page/edit.hbs

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
✂️ and use indentation.

Reply:
/edit is obsolete and was deleted

Reply:
I still can see this. Did you delete this in another PR?

Reply:
Yes, in the page-operations